Automattic/pocket-casts-android
# Use HTML in the podcast description

## Description

The server now returns a HTML formatted version of the podcast description. The podcast page has been updated to support the HTML.

There is one issue that I haven't been able to resolve. When using HTML in the view, the three dots to show that you can expand doesn't work. I’m not quite sure how to fix it yet so I will leave it for now. 

## Testing Instructions
1. Follow the podcast Dirtbag Rich https://pca.st/zkf1jxjd
2. Open the podcast page
3. Expand the header
4. Tap on the podcast description
5. ✅ Check the description is using HTML and split into paragraphs

#### Check with the feature flag off
1. Go to Profile -> Settings -> Beta features
2. Toggle off "Use HTML in the podcast description"
3. Go back to the podcast page
4. ✅ Check the description is not using HTML

## Screenshots 

| Before | After |
| --- | --- |
| ![dirtbag-rich-before](https://github.com/user-attachments/assets/32428720-975c-4485-8316-7790b1753256) | ![Screenshot_20241218_174819](https://github.com/user-attachments/assets/75195c0b-0304-40b8-830f-15198fa53632) |

## Checklist
 
#### I have tested any UI changes...
